### ISSUE TITLE
Accessibility audit: Tabs UI different from GDS Tab component

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#303]
## Describe the issue
We've replicated (with missing a11y features) the GDS Design System tab interface (which passed their a11y audit), but should we use the published Tab component for consistency?

## Related stories

- [x]  [Accessibility audit: (JAWs) Nunjucks Tab missing from list item list](https://github.com/dwp/design-system/issues/299)
- [x] [Accessibility audit: (JAWS) Tab component missing expanded states](https://github.com/dwp/design-system/issues/300)
- [x] [Accessibility audit: (JAWS) Tab component, tab differentiation with hidden text when using multiple examples](https://github.com/dwp/design-system/issues/301)
- [x] [Accessibility audit: (JAWS) Tabs appear as links but missing for link items list](https://github.com/dwp/design-system/issues/302)
